redaxo/src/addons/media_manager/install.php

```diff
@@ -11,6 +11,8 @@
  * @var rex_addon $this
  */
 
-if (!$this->hasConfig('jpg_quality')) {
+if (!$this->hasConfig()) {
     $this->setConfig('jpg_quality', 85);
+    $this->setConfig('png_compression', 6);
+    $this->setConfig('interlace', false);
```

Comment:
Je nachdem, nach welchem Kommentar man geht. 
Der eine schreibt zum Beispiel:

> I disagree entirely. If this question were about storage, this answer would be correct, but it's about the best publishing for images on the web. I would say always progressive/interlace an image since the user gets feedback about the entire image much faster than without. Users won't care about a 10% increased load time for a perfect image if they can see a decent image 60% faster.

Und da habe ich mich quasi angeschlossen mit meinem Votum für 1. ;)